static/app/views/onboarding/setupDocs.tsx

```diff
@@ -244,7 +244,9 @@ function SetupDocs({
       // If no projects remaining, then we can leave
       !project
     ) {
-      browserHistory.push('/');
+      browserHistory.push(
```

Comment:
Had to modify the URL so we can keep the referrer (it redirects to the issues list anyways)